backend/src/main/java/com/google/sps/notifiers/EmailNotifier.java

```diff
@@ -0,0 +1,49 @@
+// Class used to send users emails regarding their status in regards to their Ad-Lib Meeting.
+
+import com.sendgrid.*;
```

Comment:
I'm not exactly sure how many of the sendgrid libraries you are using, but I think it's good to have specific import statements (if that is possible).

Reply:
+1

Reply:
Fixed